js/battle.js

```diff
@@ -6745,8 +6748,6 @@ var Battle = (function () {
 				this.message('<small>' + poke.getName() + "'s disguise was busted!</small>");
 			} else if (toId(newSpecies) === 'zygardecomplete') {
 				this.message('' + poke.getName() + ' transformed into its Complete Forme!');
-			} else if (toId(newSpecies) === 'necrozmaultra') {
-				this.message('' + poke.getName() + ' regained its true power through Ultra Burst!');
```

Comment:
You can't just remove this because we still have to support old replays. Also this message happens after the animation, which is why it's separate from the one that happens prior to the animation.

Reply:
This isn't a change in protocol, though, just how the client handles it. Old replays work fine. As for the message order, this is exactly how mega evolution is handled too.

Reply:
OK in that case I got nothing. Somebody other than me with experience in rewriting client protocol will have to fix this, and while they're at it, get the right custom forme change animation to happen appropriately as well.